### PR TITLE
Stat tracker feat/database creation

### PR DESCRIPTION
**Pull request by DustinPeukert**
- wrote tests for #create_databases method
- wrote the databases method
- added missing .all method to Games class for required functionality

### CODE REVIEW

**Review by KMPWeiland — APPROVED** [review]
Nice well-rounded tests! And thanks for adding the .all method to the games class.

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
 
   def initialize(locations)
     @game_path = locations[:games]
     @team_path = locations[:teams]
     @game_teams_path = locations[:game_teams]
+    @game_teams = GameTeams
+    @teams = Teams
+    @games = Games
+
+    create_databases
```
Comment (kevin-newland) [inline lib/stat_tracker.rb:21]: Is the reason we call create_databases in the initialize method  to ensure that the necessary data is loaded and ready to use immediately upon creation?

Comment (DustinPeukert) [reply lib/stat_tracker.rb:21]: Yes it is! We want to make sure that form the moment we create our StatTracker, our 3 "database" classes are ready to go as well!

Comment (kevin-newland) [reply lib/stat_tracker.rb:21]: Thanks for the clarification!

**Review by kevin-newland — APPROVED** [review]
Looks great!

**Review by qoryhanisagal — APPROVED** [review]
I don't understand but I will catch up.

**Pull request merged**